cluster/v1beta1/types_placement.go

```diff
@@ -329,11 +364,35 @@ const (
 	TolerationOpEqual  TolerationOperator = "Equal"
 )
 
+// Present decision groups status based on the DecisionStrategy definition.
+type DecisionGroupStatus struct {
+	// Present the decision group index. If there is no decision strategy defined all placement decisions will be in group index 0
+	// +optional
+	DecisionGroupIndex int32 `json:"decisionGroupIndex"`
+
+	// Decision group name that is defined in the DecisionStrategy's DecisionGroup.
+	// +optional
+	DecisionGroupName string `json:"decisionGroupName"`
```

Comment:
I change it above to groupName instead of name/labelValue and mention that in the CRD definition  [L106](https://github.com/open-cluster-management-io/api/pull/242/files#diff-40673a7be25e1817d0abcd7d4c2c0706d9398d31b65c1c62ae5ce2f6b9521cfeR106)